extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Podcast Player Block.
+ *
+ * @since 8.x
+ *
+ * @package Jetpack
+ */
+
+namespace Jetpack\Podcast_Player_Block;
+
+use WP_Error;
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'podcast-player';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	jetpack_register_block(
+		BLOCK_NAME,
+		array(
+			'attributes'      => array(
+				'url'         => array(
+					'type' => 'url',
+				),
+				'itemsToShow' => array(
+					'type'    => 'integer',
+					'default' => 5,
+				),
+			),
```

Comment:
do you mean defining block attributes? They are needed since the client checks them through f an async request, something like ...

https://your-testing.site/wp-json/wp/v2/block-renderer/jetpack/podcast-player?context=edit&attributes%5Burl%5D=https%3A%2F%2Fdistributed.blog%2Fpodcast%2F&attributes%5BitemsToShow%5D=5&post_id=3652&_locale=user

Reply:
Removing this will cause the server-side render call to fail with a 404.

Reply:
:this: Just a minor correction. The error response is `400 (Bad request)`

Reply:
Oh good call 👍